src/db/mod.rs

```diff
@@ -386,7 +146,7 @@ impl Database {
         Ok(data)
     }
 
-    /// Get the version of a database.
+    /// Get the version of a database without loading it
```

Comment:
@sseemayer `without decrypted it`? We load it but just a bit :laughing:

Reply:
I've cleaned it up in a follow-up commit.